packages/block-editor/src/components/block-breadcrumb/test/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import BlockBreadcrumb from '../';
+
+describe( 'BlockBreadcrumb', () => {
+	it( 'should render correctly', () => {
+		const { container } = render( <BlockBreadcrumb /> );
+
+		expect( container.firstChild ).toMatchSnapshot();
+	} );
+
+	describe( 'Root label text', () => {
+		test( 'should display default label of "Document"', () => {
+			render( <BlockBreadcrumb /> );
+
+			const rootLabelTextDefault = screen.getByText( 'Document' );
+
+			expect( rootLabelTextDefault ).toBeDefined();
```

Comment:
Nitpicking:

I'm not sure about this, but I think we can use `toBeInTheDocument` here for readability.

```suggestion
			expect( rootLabelTextDefault ).toBeInTheDocument();
```

(The same goes for the other two assertions below)

Reply:
Good call. Agree 100%. 

Thanks for spotting.